io.github.khaled_0.TubeSync.yml

```diff
@@ -0,0 +1,131 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+id: io.github.khaled_0.TubeSync
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: tubesync
+finish-args:
+  - --socket=wayland # Wayland by default
+  - --socket=fallback-x11 # Fallback x11
+  - --share=ipc # X11 performance
+  - --device=dri # OpenGL rendering
+  - --socket=pulseaudio # Audio Playback
+  - --socket=system-bus # Flutter seems to complain without it
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.TubeSync.*
```

Comment:
use the appid in your code like so `org.mpris.MediaPlayer2.io.github.khaled_0.TubeSync`, then there is no need for this permission.